src/borg/fuse.py

```diff
@@ -97,6 +98,20 @@ def _create_filesystem(self):
                     self.contents[1][os.fsencode(name)] = archive_inode
                     self.pending_archives[archive_inode] = archive
 
+    def sig_info_handler(self, sig_no, stack):
+        logger.debug('fuse: %d inodes, %d synth inodes, %d edges (%s)',
+                     self._inode_count, len(self.items), len(self.parent),
+                     # getsizeof is the size of the dict itself; key and value are two small-ish integers,
+                     # which are shared due to code structure (this has been verified).
+                     format_file_size(sys.getsizeof(self.parent) + len(self.parent) * sys.getsizeof(self._inode_count)))
```

Comment:
(That's not the largest structure used by borg.fuse, just the one with the most ridiculous overhead in all dimensions)